cpp/src/gis/dispatch/dispatch.h

```diff
@@ -54,6 +54,11 @@ auto GenericArraySplit(const std::shared_ptr<TypedArrowArray>& geometries,
                        const std::vector<bool>& mask)
     -> std::array<std::shared_ptr<TypedArrowArray>, 2>;
 
+template <typename RetType, typename FalseFunc, typename TrueFunc, typename Arg1>
+auto UnaryMixedExecute(const std::vector<bool>& mask, FalseFunc false_func,
```

Comment:
UnaryMixedExecute need to be extended to support varied-length parameter list.
ie. ST_PrecisionReduce(const std::shared_ptr<arrow::Array>& geometries, int32_t precision)

Reply:
This has been in process